### ISSUE TITLE
Print old + new UID/GID when "warning: not applying GID change of group xyz"

### ISSUE THREAD

**Issue by bjornfor** [NixOS/nixpkgs#9217]
I think it would be helpful to print the old and new UID / GID numbers in the messages printed at nixos-rebuild time, as seen here:

```
$ sudo nixos-rebuild switch
...
warning: not applying GID change of group ‘munin’
warning: not applying UID change of user ‘munin’
warning: not applying UID change of user ‘ntp’
```

**Comment by Profpatsch** [comment]
How do I even fix this, do I have to manually change all files and then the UID/GID?

**Comment by lethalman** [comment]
There's nothing to fix. If you want to get rid of the warning maybe you can
override the uid of the nixos module.

On Tuesday, August 18, 2015, Profpatsch <notifications@github.com> wrote:

> How do I even fix this, do I have to manually change all files and then
> the UID/GID?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/issues/9217#issuecomment-132105053>.
>


-- 
NixOS Linux <http://nixos.org>

**Comment by Profpatsch** [comment]
If this is a warning the everyone sees and everything is okay, there should be no warning. Especially if it’s a mundane task like system profile switching.

**Comment by bjornfor** [comment]
What I meant was that instead of

```
warning: not applying GID change of group ‘munin’
```

the message could be

```
warning: not applying GID change of group ‘munin’ (UID x -> y)
```

**Comment by Profpatsch** [comment]
(triage) I’m not sure if that’s still happening. Can you construct an example that provokes the message?

**Comment by bjornfor** [comment]
@Profpatsch: Thanks for the reminder. It's fixed with commit 6e528893a8ccaf3fa4ad23bc0e4d18c679c1f37d ("nixos/update-users-groups.pl: print UIDs and GIDs"). and is backported to NixOS 16.03.
